src/cbl-mariner/2.0/crossdeps/amd64/Dockerfile

```diff
@@ -21,7 +21,9 @@ RUN tdnf update -y && \
         # Jit rolling build dependency
         python3-pip \
         # diagnostics build dependency
-        lldb-devel
+        lldb-devel \
+        powershell \
+        azure-cli
```

Comment:
I assume these aren't diagnostic build dependencies. It'd be good even these were within a comment that categorized these correctly.